workers/unity/Packages/com.improbable.gdk.core/GameObjectRepresentation/Injection/RequiresMonoBehaviourActivationManager.cs

```diff
@@ -3,7 +3,7 @@
 namespace Improbable.Gdk.Core.GameObjectRepresentation
 {
     /// <summary>
-    ///     Tag component for marking entities that require a RequiresSpatialOSBehaviourManager.
+    ///     Tag component for marking entities that require a MonoBehaviourActivationManager.
```

Comment:
Nit: I know I'm a little late to the party but would it make sense to rename `MonoBehaviourActivationManager` to `MonoBehaviourLifecycleManager`? Since it does more than activating monobehaviours?

Reply:
Ok, that sounds fair. Let's keep the current name. :)